### PR TITLE
Refactor IndexName tests

### PR DESCRIPTION
**Pull request by filipmilo**
Part of: #251

### CODE REVIEW

File: modules/library/src/test/scala/zio/elasticsearch/IndexNameSpec.scala [hunk 1]
```diff
@@ -23,51 +23,53 @@ import zio.test._
...
-          assert(IndexName.make(name))(
-            equalTo(Validation.fail(indexNameFailureMessage(name)))
-          )
-        }
-      }
+        },
+        test("fail for string containing character '*'") {
+          check(genString(0, 127), genString(0, 128)) { (part1, part2) =>
+            val invalidName = s"$part1*$part2"
+            assert(IndexName.make(invalidName))(equalTo(Validation.fail(indexNameFailureMessage(invalidName))))
```
Comment (dbulaja98) [inline modules/library/src/test/scala/zio/elasticsearch/IndexNameSpec.scala:39]: Can you put one empty line between?

File: .bsp/sbt.json
Comment (dbulaja98) [inline .bsp/sbt.json:1]: Can you remove this file?

Comment (dbulaja98) [reply .bsp/sbt.json:1]: Also .idea/misc.xml and .idea/workspace.xml.

File: modules/library/src/test/scala/zio/elasticsearch/IndexNameSpec.scala (hunk 1 above)
Comment (dbulaja98) [reply modules/library/src/test/scala/zio/elasticsearch/IndexNameSpec.scala:39]: Do this for every test.

**Review by dbulaja98 — CHANGES_REQUESTED** [review]

**Review by dbulaja98 — APPROVED** [review]

**Review by markaya — APPROVED** [review]

**Pull request merged**